modules/ROOT/pages/create-instance-task.adoc

```diff
@@ -14,17 +14,17 @@ the sandbox environment, and you want to promote it to the current or production
 * *Import API from zip file:* Select this option if you have exported an API Instance from a different Anypoint Platform
 instance.
 
-When you are promoting or importing an API Instance, you will not have options to alter the configuration. However,
-when you add a new API Instance, you will need to configure the endpoint as well as other settings.
+When you are promoting or importing an API Instance, you do not have options to alter the configuration. However,
+when you add a new API Instance, you need to xref:configure-api-task.adoc[configure the endpoint] as well as other settings.
 
 == Add a New API
 
 . Log in to Anypoint Platform.
 . In the navigation bar or the Anypoint Platform page, click *API Manager*.
 . Click *Add API* and select *Add new API*.
 . Select and configure your runtime from the following options:
-Flex Gateway:: Select Flex Gateway for any APIs that need a flexible, performant API gateway
-that will work with distributed and microservices environments and fit into your CI/CD environments.
+Flex Gateway:: Select Flex Gateway for any APIs that need a flexible and performant API gateway
+that works with distributed and microservices environments and fits into your CI/CD environments.
```

Comment:
This sentence seems to be a bit long. Maybe:
"Select Flex Gateway for any APIs that need a flexible and performant API gateway. Such API getaway should work with distributed microservices environments, and fit into your CI/CD environments."
And, is it possible that the "and" between "distributed" and "microservices" should not be there?